spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/metadata.scala

```diff
@@ -81,7 +81,9 @@ object metadata {
     AtlasTypeUtil.createUniqueRequiredAttrDef(
       "qualifiedName", new AtlasStringType),
     AtlasTypeUtil.createOptionalAttrDef("tableType", new AtlasStringType),
+    AtlasTypeUtil.createOptionalAttrDef("schemaDesc", new AtlasStringType),
     AtlasTypeUtil.createOptionalAttrDef("provider", new AtlasStringType),
+    AtlasTypeUtil.createOptionalAttrDef("partitionProvider", new AtlasStringType),
```

Comment:
@HeartSaVioR , Assume this should get added to the atlas spark model JSON files as well? Let me know if that changes should go to Atlas repo.

For the hive_table entity should we just skip these attributes for now or update the hive models and HMS hook to populate this?

Reply:
@arunmarhadevan

> Assume this should get added to the atlas spark model JSON files as well?

Yeah right. Maybe better to handle this first, and handle #243 later. cc. @aadarshjajodia You may want to track this PR as well. We'll merge this soon.

> For the hive_table entity should we just skip these attributes for now or update the hive models and HMS hook to populate this?

 I guess it's same question above - I don't think we should expand Hive models to fit to Spark catalog table's information. Let's skip these attributes.

Reply:
> @HeartSaVioR , Assume this should get added to the atlas spark model JSON files as well? Let me know if that changes should go to Atlas repo.

Yes Arun is correct. If we want to add any attribute to spark we should add it in the JSON file. Also I believe we can keep the attributes same as that of hive, unless there's something specific to spark which we might want to capture?

> For the hive_table entity should we just skip these attributes for now or update the hive models and HMS hook to populate this?

Reply:
@aadarshjajodia
There're so many difference of information what Hive and Spark are providing - that's why we defined Spark models instead of just leveraging Hive models. If we just let Spark entities to leverage Hive models, bunch of information will be just discarded.

Here's the excel file which compared Spark vs Hive models (slightly outdated due to the change of #169):
https://github.com/hortonworks-spark/spark-atlas-connector/files/3141841/SAC-168-table-models-description.copy.xlsx

Reply:
@aadarshjajodia The attributes are different so we cannot keep it same as hive.
Can you let me know if the JSON file for Spark entities are already checked into Atlas repo ? If you are yet to create it, please include the attributes defined in this PR for the `spark_table` entity.